pkg/apis/core/types.go

```diff
@@ -515,23 +515,23 @@ const (
 )
 
 // +enum
-type PersistentVolumeClaimResizeStatus string
+type ClaimResourceStatus string
 
 const (
-	// When expansion is complete, the empty string is set by resize controller or kubelet.
-	PersistentVolumeClaimNoExpansionInProgress PersistentVolumeClaimResizeStatus = ""
```

Comment:
Will the new ClaimResourceStatus still be "" if the expansion is complete?

Reply:
No it won't be. The map will be simply be set back to nil - https://github.com/kubernetes/kubernetes/pull/116335/files#diff-36cd1ab42888b4ff92b5b9855ea7282a04d855d6f9c96cb95e7867cd9d999fa1R227

Reply:
Added a comment.